packages/manager/apps/telecom/src/app/app.controller.js

```diff
@@ -26,6 +26,13 @@ export default class TelecomAppCtrl {
     this.ovhFeatureFlipping = ovhFeatureFlipping;
 
     this.chatbotEnabled = false;
+    this.SYSTRON_FEEDBACK_INFO = {
```

Comment:
:warning: There is a typo (same on commit message and Pull Request title)
```suggestion
    this.SYSTRAN_FEEDBACK_INFO = {
```